stdlib/public/TensorFlow/CompilerRuntime.swift

```diff
@@ -346,17 +351,24 @@ public final class _ExecutionContext {
       debugLog(
         "Device \(deviceId) has type \(deviceType) and name \(deviceName)."
       )
-      if deviceType == "CPU", deviceName == localCPUDeviceName {
+      if deviceType == "CPU", deviceName.starts(with: self.cpuDeviceNamePrefix) {
```

Comment:
Locally, we prefer omitting `self.` unless it's strictly necessary. In this case I think we can omit this.
```suggestion
      if deviceType == "CPU", deviceName.starts(with: cpuDeviceNamePrefix) {
```